docs/standard/linq/generate-xml-csv-files.md

```diff
@@ -0,0 +1,140 @@
+---
+title: How to generate XML from CSV files - LINQ to XML
+description: You can C# and Visual Basic to create XML from a comma-separated values (CSV) file. The example in this article shows how.
```

Comment:
I hate "Learn now" because it is an order to the reader; I realize I may be alone in this view.

- [ ]

Reply:
The description is supposed to be a call to action and feels a bit like marketing.

Here's more info on the subject: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=master

Reply:
My attitude on this has been that the best marketing would be a clear, concise, and accurate description of the article's contents. Wikipedia says "The key to a powerful call-to-action is to provide consumers with compelling reasons to purchase promptly rather than defer purchase decisions." For article descriptions, the compelling reasons are the article content. Not so? Anyway, I have never had any talent or will for exhortations. Maybe I could just continue with my descriptions, and the "calls to action" could be a later enhancement? Since most of the descriptions are blank now we will still be improving the situation (and going beyond scope. What do you think?